Update index.md

gaknoll opened the pull request:
Updated the high-level introduction to HockeyApp transition.

gaknoll committed:
Update index.md

jwargo committed:
Update index.md

jwargo commented on docs/migration/index.md:
```diff
@@ -17,7 +17,9 @@ It provides a unified suite of capabilities that will greatly benefit a mobile a
-Today customers can [sign-in](https://appcenter.ms/login?utm_medium=referral_link&utm_source=Hockey%20App) to App Center using their existing HockeyApp account credentials. We've synchronized your existing HockeyApp apps and data with App Center. This allows you to login to App Center using your HockeyApp credentials and try out all the services App Center offers without affecting your HockeyApp  ...
+App Center is the new home for the next generation of HockeyApp’s distribution, crash reporting and analytics services. We will, at some point in the future, announce plans to shut down HockeyApp and migrate all customers and their data to App Center. Prior to the announcement and during the subsequent year-long shutdown period, customers will be able to use both HockeyApp and App Center together ...
+
+We've synchronized your existing HockeyApp apps and data with App Center in what we consider our side-by-side experience. This allows you to [sign-in to App Center](https://appcenter.ms/login?utm_medium=referral_link&utm_source=Hockey%20App) today using your HockeyApp credentials, and try out all the new services App Center offers without affecting your HockeyApp workflow. Your apps are now two p ...
```
synchronicity?  Is that a word? Please consider synchronization.

gaknoll committed:
Update index.md

Updated to synchronization - I had kept "synchronicity" from what was there before. I have no idea if that is actually a word or not!

gaknoll commented:
@jwargo updated to get rid of "synchronicity"!
